app/views/restaurants/index.html.erb

```diff
@@ -1,3 +1,22 @@
+
 <% @restaurants.each do |restaurant| %>
-  <%= restaurant.inspect %>
+  <div class="col-sm-4 col-lg-4 col-md-4">
+    <div class="thumbnail transparency">
+      <div class="result-details">
+        <h4><%= restaurant.name %></h4>
+        <%= link_to 'Show', restaurant %>
```

Comment:
you can make the entire `<div>` clickable instead of a `show` link...

```erb
<%= link_to restaurant  do %>
  <div class="result-details">
    <h4><%= restaurant.name %></h4>
    ...
<% end %>
```